src/Components/Admin/Rooms/rooms.js

```diff
@@ -26,10 +31,10 @@ export default function EscapeRooms({
     return (
         <Container component={"main"} maxWidth="lg">
             <CssBaseline />
+            <Typography component="div" variant="h5" className={classes.title}>
+                Your escape rooms
+            </Typography>
```

Comment:
I believe this should be bolded (probably h4/h3) when I look at Figma

Reply:
Thanks, updated and bolded the title. h4/h3 increase in font sizes by quite a bit compared to h5, I think h5 is the closest fit
(image of h4)
![image](https://user-images.githubusercontent.com/56518575/126878133-a5ba06bb-59ed-41be-9806-af1a6660478c.png)